.env.example

```
@@ -1,58 +1,82 @@
+# DATABASE_URL=postgresql://bot_user:YOUR_PASSWORD@localhost:5432/bot_db
+#
+# Avoid `docker compose down -v` / `yarn docker:dev:down:volumes` unless you intend to wipe the database volume.
+
+# Other Credentials (if used by bot features)
+EMAIL_USER=
+EMAIL_PASS=
+# Inbox for the disabled /suggest command (nodemailer To: address), if you enable it
+FEEDBACK_EMAIL=
+
+# Lavalink Configuration
+LAVALINK_HOST=lavalink # Should match the service name in docker-compose.yml
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)